pkg/bbgo/exit.go

```diff
@@ -35,6 +37,35 @@ type ExitMethod struct {
 	TrailingStop              *TrailingStop2             `json:"trailingStop"`
 }
 
+func (e ExitMethod) String() string {
```

Comment:
since it's String(), maybe use YAML format for more readability ?

Reply:
using YAML format will occupy more spaces/lines than json.  Notice that the final output will be inside a column of a table.
also inside ExitMethod, we only define json tags, so YAML format won't work naively.